include/boost/http_proto/message_base.hpp

```diff
@@ -37,6 +37,16 @@ class message_base
     {
     }
 
+    message_base(
+        detail::kind k,
+        char* storage,
+        std::size_t storage_size) noexcept
+        : fields_view_base(&this->fields_base::h_)
+        , fields_base(
+            k, storage, storage_size)
```

Comment:
If you want to get rid of constructors should we require all constructors of `fields_base` to also have the `string_view` parameter, defaulted to the empty string?